src/docs/truffle/reference/configuration.md

```diff
@@ -51,6 +51,135 @@ Any of the following solutions will remedy this issue:
 
 ## General options
 
+### SolidityLog
+
+Like javascript's console.log, you can utilize the same concept to print state
+of your contract during execution. This log will appear when you use `truffle
+test` or `truffle develop`. To use it you would import `truffle/console.sol`
+into your solidity contract and use it like you would in JavaScript. See the 
+[Ganache documentation](https://github.com/trufflesuite/ganache/tree/develop/src/chains/ethereum/console.log#ganacheconsolelog) for details.
+
+<details><summary>Consider this solidity contract</summary>
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.25 <0.9.0;
+
+import "truffle/console.sol";
+
+contract Telephone {
+    uint public secret;
+
+    constructor(uint _secret) payable { 
```

Comment:
Is the underscore here idiomatic Solidity?

Reply:
I'm not sure what idiomatic Solidity is. I'm going to rework the example.

Reply:
This example is fine. I was just curious!